tsdb/config.go

```diff
@@ -57,7 +57,7 @@ type Config struct {
 	CompactFullWriteColdDuration   toml.Duration `toml:"compact-full-write-cold-duration"`
 	MaxPointsPerBlock              int           `toml:"max-points-per-block"`
 
-	DataLoggingEnabled bool `toml:"data-logging-enabled"`
+	TraceLoggingEnabled bool `toml:"trace-logging-enabled"`
```

Comment:
May need to update the sample config and add a note in the release notes sections.